.changeset/good-beans-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': minor
```

Comment:
Hmm this should be a `patch` changeset as the package is below version 1 and it's not technically breaking.

Reply:
Fixed.